### PR TITLE
create Db structure

### PR DESCRIPTION
**Pull request by nesclark**
create sql file and export into repo
contains data and table structure for given theme

### CODE REVIEW

File: house_plants.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `plants`;
+
+CREATE TABLE `plants` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `colloquial_name` varchar(255) DEFAULT NULL,
+  `latin_name` varchar(255) DEFAULT NULL,
+  `size_cm` int(11) DEFAULT NULL,
+  `image_url` varchar(255) DEFAULT NULL,
```
Comment (alreuben) [inline house_plants.sql]: did you want to allow nulls for all of these?

File: house_plants.sql [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# Dump of table plants
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `plants`;
+
+CREATE TABLE `plants` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `colloquial_name` varchar(255) DEFAULT NULL,
+  `latin_name` varchar(255) DEFAULT NULL,
+  `size_cm` int(11) DEFAULT NULL,
```
Comment (alreuben) [inline house_plants.sql]: could change this to unsigned so it can't be a minus value

**Review by alreuben — CHANGES_REQUESTED** [review]

File: house_plants.sql (hunk 1 above)
Comment (nesclark) [reply house_plants.sql]: thanks for highlighting this, have implemented changes

File: house_plants.sql (hunk 2 above)
Comment (nesclark) [reply house_plants.sql]: Thanks, have made changes

**Review by alreuben — APPROVED** [review]

**Pull request merged**